src/func.php

```diff
@@ -197,6 +207,8 @@ function show_help(): void {
 	. colored('--restore[=backup]', TextColor::LightGreen) . $nl
 	. "  Restore from --backup-dir. Just --restore lists available backups.$nl"
 	. "  --restore=backup will restore from <--backup-dir>/backup.$nl$nl"
+	. colored('--disable-telemetry', TextColor::LightGreen) . $nl
+	. "  Pass this flag in case you want to disable sending metrics to our servers.$nl$nl"
```

Comment:
```suggestion
	. "  Pass this flag in case you want to disable sending anonymized metrics to Manticore. You can also use environment variable TELEMETRY=0.$nl$nl"
```